packages/next-swc/crates/next-build/src/next_build.rs

```diff
@@ -0,0 +1,610 @@
+                        .try_join()
+                        .await?
+                );
+
+                let absolute_page_path = format!("/{relative_page_path}");
+
+                let build_manifest_pages_entry = build_manifest
+                    .pages
+                    .entry(absolute_page_path.clone())
+                    .or_default();
+                for chunk in client_chunks.await?.iter() {
+                    let chunk_path = chunk.ident().path().await?;
+                    if let Some(asset_path) = build_manifest_dir_path.get_path_to(&*chunk_path) {
+                        build_manifest_pages_entry.push(asset_path.to_string());
+                    }
+                }
+
+                let chunk_path = node_chunk.ident().path().await?;
+                if let Some(asset_path) = pages_manifest_dir_path.get_path_to(&*chunk_path) {
+                    pages_manifest
+                        .pages
+                        .insert(absolute_page_path, asset_path.to_string());
+                }
+
+                // for asset_path in page_asset_paths.iter() {
+                //     if let Some(asset_path) =
+                // node_root.get_path_to(&*asset_path) {
+                //         pages_manifest_entry.push(asset_path);
+                //     } else if let Some(asset_path) =
+                // server_root.get_path_to(&*asset_path) {
+                //         build_manifest_pages_entry.push(asset_path);
+                //     }
+                // }
+            }
+
+            tracing::debug!(
+                "all node assets: {}",
+                deduplicated_node_assets
+                    .values()
+                    .map(|asset| async move {
+                        Ok(format!("  - {}", asset.ident().path().to_string().await?))
+                    })
+                    .try_join()
+                    .await?
+                    .join("\n")
+            );
+            deduplicated_node_assets
+                .into_values()
+                .map(|asset| async move {
+                    emit(asset).await?;
+                    Ok(())
+                })
+                .try_join()
+                .await?;
+
+            tracing::debug!(
+                "all client assets: {}",
+                deduplicated_client_assets
+                    .values()
+                    .map(|asset| async move {
+                        Ok(format!("  - {}", asset.ident().path().to_string().await?))
+                    })
+                    .try_join()
+                    .await?
+                    .join("\n")
+            );
+            deduplicated_client_assets
+                .into_values()
+                .map(|asset| async move {
+                    emit(asset).await?;
+                    Ok(())
+                })
+                .try_join()
+                .await?;
+        }
+
+        let middleware_manifest = serde_json::to_string_pretty(&MiddlewaresManifest {
```

Comment:
Here, we create all necessary (blank) manifests for the rest of the Next.js build pipeline to work properly.